velox/exec/Exchange.h

```diff
@@ -195,6 +265,12 @@ class ExchangeQueue {
     clearPromises(promises);
   }
 
+  /// After a reply has arrived, finds other sources to request. See the same
+  /// method in ExchangeClient for the meaning of the arguments.
+  bool requestIfDue(
+      const std::shared_ptr<ExchangeSource>& replySource,
+      int64_t replySequence);
```

Comment:
replySequence is an implementation of the ExchangeSource and should not be leaked. Let's remove this parameter.